src/automation/azext_automation/tests/latest/test_automation_scenario_manual.py

```diff
@@ -210,7 +229,7 @@ def test_automation_software_update_configuration(self, resource_group):
             self.check('updateConfiguration.nonAzureComputerNames', ['nonvm1', 'nonvm2']),
             self.check('updateConfiguration.operatingSystem', 'Windows'),
             self.check('updateConfiguration.targets.azureQueries[0].locations', ['eastus', 'westus']),
-            self.check('updateConfiguration.targets.azureQueries[0].scope', ['/subscriptions/00000000-0000-0000-0000-000000000000']),
+            self.check('updateConfiguration.targets.azureQueries[0].scope', ['/subscriptions/cd45f23b-b832-4fa4-a434-1bf7e6f14a5a']),
```

Comment:
> FAIL: test_automation_software_update_configuration (azext_automation.tests.latest.test_automation_scenario_manual.AutomationScenarioTest)
.....
  File "/mnt/vss/_work/1/s/azure-cli/src/azure-cli-testsdk/azure/cli/testsdk/checkers.py", line 34, in __call__
    raise JMESPathCheckAssertionError(self._query, self._expected_result, actual_result,
azure.cli.testsdk.exceptions.JMESPathCheckAssertionError: Query 'updateConfiguration.targets.azureQueries[0].scope' doesn't yield expected value '['/subscriptions/cd45f23b-b832-4fa4-a434-1bf7e6f14a5a']', instead the actual value is '['/subscriptions/00000000-0000-0000-0000-000000000000']'. Data:

The new CI issues caused by the modifications here

Reply:
In order to avoid exposing sensitive information, the real subscription will be replaced by mock subscription `00000000-0000-0000-0000-000000000000` in recording files by testing framework, so the expected value here should be `['/subscriptions/00000000-0000-0000-0000-000000000000']`

Reply:
> self.check('updateConfiguration.targets.azureQueries[0].scope', ['/subscriptions/00000000-0000-0000-0000-000000000000']),

However, according to the original logic, the live mode of the test cannot run successfully

Reply:
Therefore, my suggestion is to dynamically obtain the subscription id by using `subscription_id = self.get_subscription_id()` instead of hard coding, so as to avoid the inconsistency between the real subscription and the mock subscription

Reply:
Thanks for suggestion, Updated accordingly